paymentsheet/src/main/java/com/stripe/android/paymentsheet/addresselement/AutocompleteScreen.kt

```diff
@@ -158,6 +158,10 @@ internal fun AutocompleteScreenUI(viewModel: AutocompleteViewModel) {
                                             .clickable {
                                                 viewModel.selectPrediction(prediction)
                                             }
+                                            .padding(
+                                                vertical = 8.dp,
+                                                horizontal = 16.dp
+                                            )
```

Comment:
Throughout this file, I made a few changes to where padding is applied. I did this to make sure that the click ripple effect looks correct, meaning that it spans the entire width and height of a row.